docs/sections/user_guide/cli/drivers/mpas.rst

```diff
@@ -0,0 +1,109 @@
+``mpas``
+==========
+
+The ``uw`` mode for configuring and running the MPAS forecast model. Any of the ``TASK`` arguments listed below may be called to generate that component required for a the ultimate ``run`` task. A ``provisioned_run_directory`` will perform all of the tasks except for running the executable.
```

Comment:
I don't think we ever really mentioned this anywhere...

I would love feedback on how I could say this more clearly.